server/students.js

```diff
@@ -49,7 +49,7 @@ router.delete('/:id', function(req,res,next){
 
 router.post('/', function(req,res,next){
   console.log(req.body)
-  Campus.findOne({
+  Campus.findOne({//Any reason you didn't just do findById?
```

Comment:
findOne is just fine.  Just wondering why you didn't do findById, which is much easier